boards/README.md

```diff
@@ -4,36 +4,79 @@ Platforms Supported by Tock
+| [SiFive HiFive1 Rev B](hifive1/README.md)                         | RISC-V           | FE310-G002     | openocd    | tockloader                  | Yes (5.1)     |
+| [ESP32-C3-DevKitM-1](esp32-c3-devkitM-1/README.md)                | RISC-V-ish RV32I | ESP32-C3       | custom     | custom                      | No            |
 | [i.MX RT 1052 Evaluation Kit](imxrt1050-evkb/README.md)           | ARM Cortex-M7    | i.MX RT 1052   | custom     | custom                      | No            |
 | [Teensy 4.0](teensy40/README.md)                                  | ARM Cortex-M7    | i.MX RT 1062   | custom     | custom                      | No            |
-| [Pico Explorer Base](pico_explorer_base/README.md)                | ARM Cortex-M0+   | RP2040         | openocd    | openocd                     | No            |
-| [Raspberry Pi Pico](raspberry_pi_pico/README.md)                  | ARM Cortex-M0+   | RP2040         | openocd    | openocd                     | No            |
-| [SiFive HiFive1 Rev B](hifive1/README.md)                         | RISC-V           | FE310-G002     | openocd    | tockloader                  | Yes (5.1)     |
 | [Digilent Arty A-7 100T](arty_e21/README.md)                      | RISC-V RV32IMAC  | SiFive E21     | openocd    | tockloader                  | No            |
-| [OpenTitan Earlgrey on CW310](opentitan/earlgrey-cw310/README.md) | RISC-V RV32IMC   | EarlGrey       | custom     | custom                      | Yes (5.1)     |
+
+
+### Tier Z
+
+| Board                                                             | Architecture     | MCU            | Interface  | App deployment              | QEMU Support? |
+|-------------------------------------------------------------------|------------------|----------------|------------|-----------------------------|---------------|
+| [QEMU RISC-V 32 bit `virt` platform](qemu_rv32_virt/README.md)    | RISC-V RV32IMAC  | QEMU           | custom     | custom                      | Yes (duh)     |
 | [LiteX on Digilent Arty A-7](litex/arty/README.md)                | RISC-V RV32IMC   | LiteX+VexRiscV | custom     | tockloader (flash-file)[^1] | No            |
 | [Verilated LiteX Simulation](litex/sim/README.md)                 | RISC-V RV32IMC   | LiteX+VexRiscv | custom     | tockloader (flash-file)[^1] | No            |
-| [ESP32-C3-DevKitM-1](esp32-c3-devkitM-1/README.md)                | RISC-V-ish RV32I | ESP32-C3       | custom     | custom                      | No            |
-| [QEMU RISC-V 32 bit `virt` platform](qemu_rv32_virt/README.md)    | RISC-V RV32IMAC  | QEMU           | custom     | custom                      | Yes (duh)     |
+| [OpenTitan Earlgrey on CW310](opentitan/earlgrey-cw310/README.md) | RISC-V RV32IMC   | EarlGrey       | custom     | custom                      | Yes (5.1)     |
```

Comment:
This is one of the most used platforms and is part of the CI. I think it should be tier1 not experimental

Reply:
I think that _experimental_ here really does mean that these board definitions are -- by their nature -- moving targets. As soon as there is a physical chip of OpenTitan with reasonable availability, I suppose that very few would be averse to classifying it as a Tier 1 board.

Nonetheless, the term "experimental" does sound like there's an expectation of instability associated with these boards. I might agree that OpenTitan is one of our more well-tested boards, and the `litex-sim-ci` workflow should make sure that the few features which the LiteX boards do support work quite reliably. I'm having a hard time coming up with a better name though.

Reply:
I tried splitting into tier Z1/Z2? I'm not sure I love the language or the binning I made though..

Reply:
From the description of tier Z:

> Tock bringup is happening in parallel with hardware development

That perfectly describes Earlgrey right now. As long as you have to pin to a particular OT commit to make this port work, I don't think it should be tier 1.

Reply:
> I tried splitting into tier Z1/Z2?

Did you push that? I'm not sure whether I'd be a fan of that. I think that for us, Tier 1/2/3/Z is already relatively granular, I fear with Z1/Z2 we'd get into pointless discussions about what it actually means to be more or less experimental. Is it stability of the hardware, stability of Tock given some pinned hardware revision, stability of the composite moving targets of Tock + hardware...?

Reply:
It's here now if you look at the rendered view: https://github.com/tock/tock/blob/6b28bc6414e588c0d74dc50dea0568ae5d7ec56b/boards/README.md

I agree with your concern Leon, but I think practically it looks fine. Especially with the descriptions Pat put under them.

Reply:
I suggest putting the clue_nrf52840 in tier 1 as it is fully supported and work with tockloader and tockbootloader. We still use the board for the display.

Reply:
Clue moved to tier 1.

Are we happy with the Z1/Z2 stuff? I'm open to whatever, but I think this is approaching best-first-cut that we can refine if/when it proves necessary?

Reply:
> Are we happy with the Z1/Z2 stuff?

I'm not a fan for the reasons outlined above, but also not averse to this. I think it looks fine with the descriptions for Z1 & Z2.